### ISSUE TITLE
The timezone of the times displayed on the transactions list page is not clear

### ISSUE THREAD

**Issue by shendy-a8c** [Automattic/woocommerce-payments#9472]
### Describe the bug

The times displayed on the transactions list page under the `Date / Time` column is in store's timezone (see this [line](https://github.com/Automattic/woocommerce-payments/blob/30a1d966e598782961f3e35c3b3ac5ac67c1b412/client/transactions/list/index.tsx#L467-L470)). This is not clearly stated.

A possible solution is to add the timezone in the column header, eg `Date / Time (UTC+3)`, either by:
- Passing `wp_timezone_string()` from PHP via `wcpaySettings` and render it, or
- Just in Javascript, call `dateI18n()`:
```
dateI18n('TP') // Will display UTC+01:00
```

### To Reproduce

1. Set blog timezone to be different from browser's timezone. wp-admin > Settings > General > Timezone.
2. Buy something to create a transaction.
3. See the time of transaction is displayed in store's timezone.

### Actual behavior

It's not clear which timezone the transaction time is displayed in. Intuitively, user might think it's in their local / browser's timezone but actually it is not.

### Expected behavior

It's clear to user which timezone the transaction time is displayed in.

**Comment by haszari** [comment]
I think the root cause of this bug is that the transactions report is not in UTC, all others are. See https://github.com/Automattic/woocommerce-payments-server/issues/2847

- https://github.com/Automattic/woocommerce-payments-server/issues/2275
- https://github.com/Automattic/woocommerce-payments/pull/4711

In that issue and PR, the bug was that the time/date filtering was not consistent. The fix was to localise the times in the list view UI and the filters. 

However, I think that introduced a different bug: transactions list view and CSV are now sometimes in localised timezone, when all other reports (and server-generated transactions CSV) are in UTC.

I think we should close this and fix the underlying bug:

- https://github.com/Automattic/woocommerce-payments-server/issues/2847

**Comment by haszari** [comment]
> The times displayed on the transactions list page under the Date / Time column is in store's timezone (see this [line](https://github.com/Automattic/woocommerce-payments/blob/30a1d966e598782961f3e35c3b3ac5ac67c1b412/client/transactions/list/index.tsx#L467-L470)). This is not clearly stated.

> A possible solution is to add the timezone in the column header, eg Date / Time (UTC+3), either by:

I don't think we should add timezone labels. Long term, this would have 2 risks:

- Clutter the UI.
- Reduce trust in the UI – if we are labelling things individually, suggests we are inconsistent.

I think we need to review all UI with dates & times and decide what timezone(s) should be used where. This can guide decisions/designs, be documented clearly, and if there are edge cases we can use labels in those scenarios (e.g. if we show one specific date in UTC for some reason).

**Comment by haszari** [comment]
Marking as `blocked` until we have audited and aligned on a path forward:

- Automattic/woocommerce-payments-server#6646

**Comment by flakas** [comment]
Closing this — the underlying concern is now covered by #9184, and the specific fix proposed here has been ruled out deliberately.
The new agreed standard is to show everything in the store's configured time zone.
